### PR TITLE
tostr staticmethod error (closes #373)

### LINKED ISSUE #373: Remove staticmethod for tostr() in Stmt_Function_Stmt class
A spurious staticmethod decorator has been added to the tostr() method in the Stmt_Function_Stmt class in Fortran2003.py.

### PR DESCRIPTION
**Pull request by rupertford**

### CODE REVIEW

**Comment by rupertford** [comment]
Ready for review.

File: src/fparser/two/Fortran2003.py [hunk 1]
```diff
@@ -12060,7 +12060,6 @@ def match(string):
             return Function_Name(name), Dummy_Arg_Name_List(args), Scalar_Expr(expr)
         return Function_Name(name), None, Scalar_Expr(expr)
 
-    @staticmethod
     def tostr(self):
```
Comment (arporter) [inline src/fparser/two/Fortran2003.py:12063]: Presumably we're missing a test for this routine. Please could you add one.

**Review by arporter — CHANGES_REQUESTED** [review]
We need a test to demonstrate the problem that has been fixed. Once that's added then this is fine as it's a simple bug fix.

File: src/fparser/two/Fortran2003.py (hunk 1 above)
Comment (rupertford) [reply src/fparser/two/Fortran2003.py:12063]: Added test file.

**Review by arporter — APPROVED** [review]
All good now and you've bumped up the test coverage which is pleasing :-)

**Pull request merged**